dlerror.go

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023 The Ebitengine Authors
 
+//go:build darwin || linux
```

Comment:
This actually removes all platforms except Darwin and Linux. Shouldn't it be `!windows` to just remove windows.

Reply:
Dl* is supported only by darwin or linux. If we want to use `!windows`, we should update all go:build for consistency. What do you think?